docs/extend/modular-resources/examples/csi.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Add a CSI Camera as a Modular Resource"
+linkTitle: "CSI Camera"
+weight: 40
+type: "docs"
+description: "How to add a CSI Camera as a modular resource of your robot."
+tags: ["board", "csi", "jetson", "serial", "module", "modular resources", "Python", "python SDK", "nvidia", "jetson orin", "jetson orin nano", "nano", "camera"]
+# SMEs: Sean
+---
+
+
+Viam provides a modular resource [extending](/extend/modular-resources/) the [camera API](/components/camera/#api) as a new `viam:camera:csi` model of [camera](/components/camera/).
+
+This module includes a simple wrapper around `GStreamer` and a control interface for the **control** tab of the [Viam app](https://app.viam.com) so you can utilize the hardware accelerated GST plugins and use the embedded CSI cameras on your `jetson` boards with Viam.
+
+The module is open-sourced and available on [GitHub](https://github.com/seanavery/viam-csi).
+
+[Install requirements](#requirements) and [configure](#configuration) the module to add an `viam:camera:csi` [camera](/components/camera/) {{< glossary_tooltip term_id="resource" text="resource" >}} to your robot.
```

Comment:
missing article in `Install requirements` but that also doesn't sound super great? Should that be "get the modular resouce" or something like that?

Reply:
Hmmm I can change here but I don't know if we actually need to change the beginning of the sentence! I think that without article is actually the normal way to say it, but I could be wrong 😆 Requirements as a noun and directive, right? Like a medium article?